doc/release-notes/release-5.sgml

```diff
@@ -12,13 +12,11 @@ for Applied Network Research and members of the Web Caching community.
 <toc>
 
 <sect>Notice
-<p>The Squid Team are pleased to announce the release of Squid-5.0.7 for testing.
+<p>The Squid Team are pleased to announce the release of Squid-5.1.
 
 This new release is available for download from <url url="http://www.squid-cache.org/Versions/v5/"> or the
  <url url="http://www.squid-cache.org/Download/http-mirrors.html" name="mirrors">.
```

Comment:
Consider removing a reference to mirrors, assuming we are deprecating them due to TLS-everywhere complications.

```suggestion
This new release is available for download from <url url="http://www.squid-cache.org/Versions/v5/">.
```

Reply:
The TLS-everywhere situation does not affect the relevance of http-mirrors.html listing. The links from that page will just change as-needed, which is why we list it here instead of the links themselves.

Reply:
I bet there will be no mirrors after we switch to HTTPS.